packages/vaadin-themable-mixin/lumo-injection-mixin.js

```diff
@@ -57,8 +57,10 @@ export const LumoInjectionMixin = (superClass) =>
       }
     }
 
-    static get lumoInjectPropName() {
-      return `--${this.is}-lumo-inject`;
```

Comment:
Why was this property removed? The value is still used in two different files, but now `LumoInjector` has to construct the key "manually".